README.md

```diff
@@ -1,19 +1,49 @@
 # Brooklyn CLI
 
-## Compiling
-
-1. Ensure your [$GOPATH](http://golang.org/cmd/go/#hdr-GOPATH_environment_variable) is set correctly,
-   to some location where Go does its work, such as `~/gocode` .
-2. Get and build the cli source code: `go get github.com/brooklyncentral/brooklyn-cli/br`
-3. Run it from `$GOPATH/bin/br` (add `$GOPATH/bin/` to your PATH).
-4. Thereafter if you want to do code changes, 
-   link the `$GOPATH/src/github.com/brooklyncentral/brooklyn-cli`
-   with the directory where you want to keep your git repositories.
-   (TODO: clarify best practice for this, including how to combine
-   it with a Brooklyn all-projects build)
+## Toolchain
+
+You will need the following tools to build the CLI:
+- Go (min version 1.5.1), with full cross-compiler support (see https://golang.org/doc/install/source).
+  On Mac, if using Homebrew, use "brew install go --with-cc-all"
+- godep (see https://github.com/tools/godep)
+
+Optional:
+- Maven (used by the Brooklyn build process)
+
+
+## Build Pre-Requisites
+
```

Comment:
Info on installing tools here:
- Install Go (`https://golang.org/dl/`)  
  On a Mac, you use for example Homebrew (`brew install go --with-cc-all`)
- Set GOPATH environment variable
- Download and install Godep  
  `go get github.com/tools/godep`
- Install Brooklyn CLI and dependencies  
  `go get -d o get github.com/brooklyncentral/brooklyn-cli/br`